po/ja.po

```diff
@@ -8224,6 +8299,8 @@ msgstr ""
 #: src/traits/default-methods.md:3
 msgid "Traits can implement behavior in terms of other trait methods:"
 msgstr ""
+"トレイトを用いて、異なるトレイトメソッドに関した動作を実装することができま"
+"す："
```

Comment:
This sentence didn't make sense to me. It just means, for a specific method of a trait, we can use its default method or implement "other" defined traits methods yourself?